templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,17 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 48**
+
+* [`POST /register`](/api/register-queue): Added `giphy_rating`
+  and `giphy_rating_options` fields.
+
+  `giphy_rating` field describes the maximum rating
+   of GIFs that should be retrieved from GIPHY.
+
+  `giphy_rating_options` field contains the valid options for
+  `giphy_rating` that can be set by organization admin.
```

Comment:
Generally, this log is intended to only tell you what changes happen, not describe the details of the changes, so I think these second two paragraphs should be removed.  

Also, the field is `giphy_rating`, not `realm_giphy_rating`, right?  And I think this commit changes another API endpoint as well, to be able to set the new field?

Reply:
Okay done. It is `giphy_rating` when doing PATCH /realm and we receive `realm_giphy_rating` from /register.